src/control/actor/host/notify.go

```diff
@@ -60,14 +60,28 @@ func (c *HostNotifyCmd) connectedF(net network.Network, conn network.Conn) {
 
 	// request metadata as soon as we connect to a peer
```

Comment:
I think we should add timestamps of both the metadata and the beacon status. Now its not very clear when we got that information. This can go in another PR but though but since it quite related I would mind having it in this one.